lib/api/core/models/repositories/tokenRepository.js

```diff
@@ -140,7 +140,7 @@ module.exports = function (kuzzle) {
     try {
       jwt.verify(token, kuzzle.config.jsonWebToken.secret);
 
-      this.load(token)
+      Repository.prototype.load.call(this, token)
```

Comment:
nope, only when we call the parent's method under the child's method, like there:
https://github.com/kuzzleio/kuzzle/blob/develop/lib/api/core/models/repositories/userRepository.js#L76
```
UserRepository.prototype.hydrate = function (user, data) {
  // (...)
  return Repository.prototype.hydrate(user, data)
  // (...)
}
```


But here, we want to call the `load` method of the current prototype (which inherits from the parent one, I know, but could be overriden), like there:
https://github.com/kuzzleio/kuzzle/blob/develop/lib/api/core/models/repositories/profileRepository.js#L38
```
ProfileRepository.prototype.loadProfile = function (profile) {
  // (...)
  return this.load(profileId)
  // (...)
}
```